code/modules/antagonists/ert/ert.dm

```diff
@@ -189,3 +194,99 @@
 
 	missiondesc += "<BR><B>Your Mission</B> : [ert_team.mission.explanation_text]"
 	to_chat(owner,missiondesc)
+
+
+/datum/antagonist/ert/families
+	name = "Space Police Responder"
+	antag_hud_type = ANTAG_HUD_SPACECOP
+	antag_hud_name = "hud_spacecop"
+
+/datum/antagonist/ert/families/apply_innate_effects(mob/living/mob_override)
+	..()
+	var/mob/living/M = mob_override || owner.current
+	add_antag_hud(antag_hud_type, antag_hud_name, M)
+
+
+/datum/antagonist/ert/families/remove_innate_effects(mob/living/mob_override)
+	var/mob/living/M = mob_override || owner.current
+	remove_antag_hud(antag_hud_type, M)
+	..()
+
+/datum/antagonist/ert/families/greet()
+	to_chat(owner, "<B><font size=3 color=red>You are the [name].</font></B>")
```

Comment:
should it be "a" or "the"? which one is more consistent with the greeting text that other ERTs and/or midround antag spawns have?

Reply:
usually if it's a lone antagonist it's the (You are the traitor.) and if it's a team antagonist it's a/an (You are a servant of Ratvar.)